autoscheduler/frontend/src/App.tsx

```diff
@@ -1,22 +1,20 @@
 import * as React from 'react';
-import { Router, Link } from '@reach/router';
-import { AppBar, IconButton, Toolbar, Button } from '@material-ui/core';
-import HomeIcon from '@material-ui/icons/Home';
+import { Router } from '@reach/router';
 import { ThemeProvider } from '@material-ui/styles';
 
 import theme from './theme';
 import Empty from './components/Empty';
-import NavBar from './Components/NavBar';
-import HelpText from './Components/HelpText';
-import SelectTerm from './Components/SelectTerm';
+import NavBar from './components/NavBar';
+import HelpText from './components/HelpText';
+import SelectTerm from './components/SelectTerm';
 
 const App: React.SFC = function App() {
   return (
     <div>
       <ThemeProvider theme={theme}>
-        <NavBar/>
-        <HelpText/>
-        <SelectTerm/>
+        <NavBar />
+        <HelpText />
+        <SelectTerm />
```

Comment:
Instead of it being here, make a `LandingPage` component in place of the `Empty` one and put it in the `Router`. This way when we have course selection we can switch between the two easily